src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
 public interface FileReaderService {
```

Comment:
Convention is to use Component on the classes that are used for instance creation (usually these are interface implementations)